README.md

```diff
@@ -2,6 +2,7 @@
 
 My personal portfolio.
 
+- Link to its online version: https://sebastiansolisfenger.github.io/my-portfolio-microverse/
```

Comment:
You have done a very good job! Congratulations :clap: :1st_place_medal: 

Just please do some adjustments to the README file in order to have a professional one, keeping in mind that first thing that a user sees on our repo is the README.

-  Kindly make sure that you respect the README template from Microverse regarding the following:
- [ ] Place the link to your project in the Live Demo Section of the template;
- [ ] Provide clear steps for a user to replicate your project on their system;
- [ ] Bring back all the missing sections of the template;

Microverse README template is here: [Microverse README TEMPLATE](https://github.com/microverseinc/readme-template)